packages/api-explorer/src/lib/get-path-operation.js

```diff
@@ -1,7 +1,7 @@
 const getPath = require('./get-path');
 
 module.exports = function getPathOperation(swagger, doc) {
-  if (swagger.paths && doc.swagger) {
+  if (swagger.paths && doc.swagger && getPath(swagger, doc)) {
```

Comment:
Can you set `getPath(swagger, doc)` to a constant and use that here and the line below?

Reply:
Done!